src/main/java/net/atos/client/zgw/zrc/model/Rol.java

```diff
@@ -103,13 +104,14 @@ public Rol() {
     /**
      * Constructor with required attributes for POST and PUT requests
      */
-    public Rol(final URI zaak, final URI roltype, final BetrokkeneType betrokkeneType, final BETROKKENE_IDENTIFICATIE betrokkeneIdentificatie,
+    public Rol(final URI zaak, final Roltype roltype, final BetrokkeneType betrokkeneType, final BETROKKENE_IDENTIFICATIE betrokkeneIdentificatie,
             final String roltoelichting) {
         this.zaak = zaak;
         this.betrokkeneIdentificatie = betrokkeneIdentificatie;
         this.betrokkeneType = betrokkeneType;
-        this.roltype = roltype;
+        this.roltype = roltype.getUrl();
         this.roltoelichting = roltoelichting;
+        this.omschrijving = roltype.getOmschrijving();
```

Comment:
Zou je dan omschrijvingGeneriek niet ook moeten overnemen uit Roltype? In commenrtaar bij veld omschrijvingGeneriek staat, net als bij veld, omschrijving: afgeleid uit het ROLTYPE.

Reply:
zal ik ook overnemen